src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-sqs-monitoring-fix

```diff
@@ -94,7 +94,7 @@ This integration collects the following Amazon SQS metrics. For full metric desc
 
     <tr>
       <td>
-        `approximateNumberOfMessagesVisible`
```

Comment:
why is the first letter capitalized; that isn't our usual standard and none of the other ones in here follow that pattern